ccloud/docs/ccloud-stack.rst

```diff
@@ -30,8 +30,33 @@ Caution
 =======
 
 This utility uses real |ccloud| resources.
+It is intended to be a quick way to create resources in |ccloud| with correct credentials and permissions, useful as a starting point from which you can then use for learning, extending, and building other examples.
+If you just run ``ccloud-stack`` without enabling |ccloud| ksqlDB, then there is no billing charge until you produce data to the |ak| cluster or provision any other fully-managed services.
+If you run ``ccloud-stack`` with enabling |ccloud| ksqlDB, then you will begin to accrue charges immediately.
```

Comment:
These 3 lines are great!

Reply:
Excellent, thanks for reviewing @matt-mangia